.github/workflows/flaky-test-detection.yml

```
@@ -0,0 +1,284 @@
+            *This issue was automatically created by the flaky test detection workflow.*
+            *Run time: ${new Date().toISOString()}*
+            `;
+            
+            // Check if there's already an open issue
+            const issues = await github.rest.issues.listForRepo({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              labels: 'flaky-test',
+              state: 'open'
+            });
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)